### ISSUE TITLE
web-ext installation not possible

### ISSUE THREAD

**Issue by PjotrC** [mozilla/web-ext#933]
### Is this a bug or feature request?
bug request

### What is the current behavior?
steps to reproduce:
1. I installed node.js
2. I don't know if this matters: I installed jpm with "npm install jpm -g" - that worked well.
3. Attempt to install web-ext with following command: "npm install --global web-ext" - this produced following output:

```
npm ERR! git clone --template=C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror git@github.com:entequak/eslint-plugin-no-unsafe-innerhtml.git C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\git-github-com-entequak-eslint-plugin-no-unsafe-innerhtml-git-4b6b606d50-e6bb2d11: undefined
npm ERR! git clone --template=C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror git@github.com:entequak/eslint-plugin-no-unsafe-innerhtml.git C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\git-github-com-entequak-eslint-plugin-no-unsafe-innerhtml-git-4b6b606d50-e6bb2d11: undefined
npm ERR! Windows_NT 10.0.15063
npm ERR! argv "D:\\nodejs\\node.exe" "D:\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "--global" "web-ext"
npm ERR! node v7.9.0
npm ERR! npm  v4.2.0
npm ERR! code ENOGIT
npm ERR! not found: git
npm ERR!
npm ERR! Failed using git.
npm ERR! This is most likely not a problem with npm itself.
npm ERR! Please check if you have git installed and in your PATH.
npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\PjotrC\AppData\Roaming\npm-cache\_logs\2017-05-01T14_48_32_486Z-debug.log
```

### What is the expected or desired behavior?
Installation should work.

### Version information (for bug reports)

* **Firefox version**: 53.0 (32-Bit)
* **Your OS and version**: Windows 10 Version 1703 (Build 15063.138)
* "node --version" output: v7.9.0
* "npm --version" output: 4.2.0
* "web-ext --version" output: 'Der Befehl "web-ext" ist entweder falsch geschrieben oder konnte nicht gefunden werden.' (means: 'The command "web-ext" is spelled wrong or could not be found')

[2017-05-01T14_48_32_486Z-debug.zip](https://github.com/mozilla/web-ext/files/968653/2017-05-01T14_48_32_486Z-debug.zip)

**Comment by kumar303** [comment]
Hi, sorry about that. It looks like it's running into a problem because git is not installed on your machine (or is it there but not on the Windows path?).

Either way, this is not documented as a requirement and it would be nice if git wasn't required at all. It looks like a recent change to the linter introduced this.

**Comment by PjotrC** [comment]
I installed git now - but I still facing problems:

```
npm WARN addRemoteGit Error: not found: git
npm WARN addRemoteGit     at getNotFoundError (D:\nodejs\node_modules\npm\node_modules\which\which.js:13:12)
npm WARN addRemoteGit     at F (D:\nodejs\node_modules\npm\node_modules\which\which.js:68:19)
npm WARN addRemoteGit     at E (D:\nodejs\node_modules\npm\node_modules\which\which.js:80:29)
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\which\which.js:89:16
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\which\node_modules\isexe\index.js:44:5
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\which\node_modules\isexe\windows.js:29:5
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\graceful-fs\polyfills.js:284:29
npm WARN addRemoteGit     at FSReqWrap.oncomplete (fs.js:114:15)
npm WARN addRemoteGit  entequak/eslint-plugin-no-unsafe-innerhtml#4b6b606d50 resetting remote C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\git-github-com-entequak-eslint-plugin-no-unsafe-innerhtml-git-4b6b606d50-8110cc66 because of error: { Error: not found: git
npm WARN addRemoteGit     at getNotFoundError (D:\nodejs\node_modules\npm\node_modules\which\which.js:13:12)
npm WARN addRemoteGit     at F (D:\nodejs\node_modules\npm\node_modules\which\which.js:68:19)
npm WARN addRemoteGit     at E (D:\nodejs\node_modules\npm\node_modules\which\which.js:80:29)
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\which\which.js:89:16
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\which\node_modules\isexe\index.js:44:5
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\which\node_modules\isexe\windows.js:29:5
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\graceful-fs\polyfills.js:284:29
npm WARN addRemoteGit     at FSReqWrap.oncomplete (fs.js:114:15) code: 'ENOGIT' }
npm ERR! git clone --template=C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror git://github.com/entequak/eslint-plugin-no-unsafe-innerhtml.git C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\git-github-com-entequak-eslint-plugin-no-unsafe-innerhtml-git-4b6b606d50-8110cc66: undefined
npm ERR! git clone --template=C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror git://github.com/entequak/eslint-plugin-no-unsafe-innerhtml.git C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\git-github-com-entequak-eslint-plugin-no-unsafe-innerhtml-git-4b6b606d50-8110cc66: undefined
npm WARN addRemoteGit Error: not found: git
npm WARN addRemoteGit     at getNotFoundError (D:\nodejs\node_modules\npm\node_modules\which\which.js:13:12)
npm WARN addRemoteGit     at F (D:\nodejs\node_modules\npm\node_modules\which\which.js:68:19)
npm WARN addRemoteGit     at E (D:\nodejs\node_modules\npm\node_modules\which\which.js:80:29)
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\which\which.js:89:16
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\which\node_modules\isexe\index.js:44:5
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\which\node_modules\isexe\windows.js:29:5
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\graceful-fs\polyfills.js:284:29
npm WARN addRemoteGit     at FSReqWrap.oncomplete (fs.js:114:15)
npm WARN addRemoteGit  entequak/eslint-plugin-no-unsafe-innerhtml#4b6b606d50 resetting remote C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\git-https-github-com-entequak-eslint-plugin-no-unsafe-innerhtml-git-4b6b606d50-637995a3 because of error: { Error: not found: git
npm WARN addRemoteGit     at getNotFoundError (D:\nodejs\node_modules\npm\node_modules\which\which.js:13:12)
npm WARN addRemoteGit     at F (D:\nodejs\node_modules\npm\node_modules\which\which.js:68:19)
npm WARN addRemoteGit     at E (D:\nodejs\node_modules\npm\node_modules\which\which.js:80:29)
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\which\which.js:89:16
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\which\node_modules\isexe\index.js:44:5
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\which\node_modules\isexe\windows.js:29:5
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\graceful-fs\polyfills.js:284:29
npm WARN addRemoteGit     at FSReqWrap.oncomplete (fs.js:114:15) code: 'ENOGIT' }
npm ERR! git clone --template=C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror https://github.com/entequak/eslint-plugin-no-unsafe-innerhtml.git C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\git-https-github-com-entequak-eslint-plugin-no-unsafe-innerhtml-git-4b6b606d50-637995a3: undefined
npm ERR! git clone --template=C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror https://github.com/entequak/eslint-plugin-no-unsafe-innerhtml.git C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\git-https-github-com-entequak-eslint-plugin-no-unsafe-innerhtml-git-4b6b606d50-637995a3: undefined
npm WARN addRemoteGit Error: not found: git
npm WARN addRemoteGit     at getNotFoundError (D:\nodejs\node_modules\npm\node_modules\which\which.js:13:12)
npm WARN addRemoteGit     at F (D:\nodejs\node_modules\npm\node_modules\which\which.js:68:19)
npm WARN addRemoteGit     at E (D:\nodejs\node_modules\npm\node_modules\which\which.js:80:29)
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\which\which.js:89:16
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\which\node_modules\isexe\index.js:44:5
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\which\node_modules\isexe\windows.js:29:5
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\graceful-fs\polyfills.js:284:29
npm WARN addRemoteGit     at FSReqWrap.oncomplete (fs.js:114:15)
npm WARN addRemoteGit  entequak/eslint-plugin-no-unsafe-innerhtml#4b6b606d50 resetting remote C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\git-github-com-entequak-eslint-plugin-no-unsafe-innerhtml-git-4b6b606d50-e6bb2d11 because of error: { Error: not found: git
npm WARN addRemoteGit     at getNotFoundError (D:\nodejs\node_modules\npm\node_modules\which\which.js:13:12)
npm WARN addRemoteGit     at F (D:\nodejs\node_modules\npm\node_modules\which\which.js:68:19)
npm WARN addRemoteGit     at E (D:\nodejs\node_modules\npm\node_modules\which\which.js:80:29)
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\which\which.js:89:16
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\which\node_modules\isexe\index.js:44:5
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\which\node_modules\isexe\windows.js:29:5
npm WARN addRemoteGit     at D:\nodejs\node_modules\npm\node_modules\graceful-fs\polyfills.js:284:29
npm WARN addRemoteGit     at FSReqWrap.oncomplete (fs.js:114:15) code: 'ENOGIT' }
npm ERR! git clone --template=C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror git@github.com:entequak/eslint-plugin-no-unsafe-innerhtml.git C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\git-github-com-entequak-eslint-plugin-no-unsafe-innerhtml-git-4b6b606d50-e6bb2d11: undefined
npm ERR! git clone --template=C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror git@github.com:entequak/eslint-plugin-no-unsafe-innerhtml.git C:\Users\PjotrC\AppData\Roaming\npm-cache\_git-remotes\git-github-com-entequak-eslint-plugin-no-unsafe-innerhtml-git-4b6b606d50-e6bb2d11: undefined
npm ERR! Windows_NT 10.0.15063
npm ERR! argv "D:\\nodejs\\node.exe" "D:\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "--global" "web-ext"
npm ERR! node v7.9.0
npm ERR! npm  v4.2.0
npm ERR! code ENOGIT

npm ERR! not found: git
npm ERR!
npm ERR! Failed using git.
npm ERR! This is most likely not a problem with npm itself.
npm ERR! Please check if you have git installed and in your PATH.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\PjotrC\AppData\Roaming\npm-cache\_logs\2017-05-03T20_27_34_039Z-debug.log
```

**Comment by kumar303** [comment]
This has been fixed in `addons-linter` 0.19.2 which is about land in web-ext; see https://github.com/mozilla/web-ext/pull/944
